packages/pn-pa-webapp/cypress/e2e/NewNotificationAdministratorWithGroups.cy.ts

```diff
@@ -0,0 +1,82 @@
+import { cesare } from '../fixtures/recipients/recipients';
+import {CREATE_NOTIFICATION} from '../../src/api/notifications/notifications.routes';
+import { PNRole } from '../../src/models/user';
+import { User } from '../../src/redux/auth/types';
+describe("Notifications New Notification", () => {
+  const pdfTest1 = './cypress/fixtures/attachments/pdf_test_1.pdf';
+  const pdfTest2 = './cypress/fixtures/attachments/pdf_test_2.pdf';
+
+  beforeEach(() => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+      return false;
+    });
+    cy.viewport(1920, 1080);
+
+    cy.logout();
+    cy.loginWithTokenExchange();    
+
+    // intercepts send notification request stubbing its successful response
+    cy.intercept('POST', CREATE_NOTIFICATION(), {
+      statusCode: 202,
+      body: {
+        notificationRequestId: 'S0FOQS1VV1JBLUFLSkUtMjAyMjExLVAtMQ==',
+        paProtocolNumber: 'prot-12555514'
+      },
+    }).as('saveNewNotification');
+    
+    cy.intercept('/delivery/attachments/preload').as('preloadAttachments');
+    
+    cy.visit('/dashboard/nuova-notifica');
+  });
+
+  it.only('Create new notification with groups and administrator role', () => {
+
+    cy.window().then((win) => {
+      let user: User = JSON.parse(win.sessionStorage.getItem('user'));
+      user.organization.roles[0].role = PNRole.ADMIN;
+      win.sessionStorage.setItem('user', JSON.stringify(user));
+    });
+
+    cy.intercept(/groups/, { fixture: 'groups/groups' });
+    
+    // Fill step 1
+    cy.fillPreliminaryInfo({
+      paProtocolNumber: 'prot-123456789',
+      subject: 'Cypress Test - Create new notification',
+      abstract: 'Testing as administrator and with user groups',
+      taxonomyCode: '012345X',
+      communicationType: 'Model_890',
+      paymentMethod: 'pagoPA'
+    });
```

Comment:
At this point you should verify the "Next" button to be disabled but this fix is not implemented yet (Jira [PN-2198](https://pagopa.atlassian.net/browse/PN-2198) is in "blocked" state). Adjust the code to verify this behaviour and comment it out saying we are waiting for the corresponding fix.